projects/plugins/jetpack/json-endpoints/jetpack/class.jetpack-json-api-sync-endpoint.php

```diff
@@ -377,9 +472,9 @@ protected function result() {
 			return new WP_Error( 'missing_item_ids', 'Please provide a list of item ids in the item_ids argument', 400 );
 		}
 
-		//Limit to A-Z,a-z,0-9,_,-
+		// Limit to A-Z,a-z,0-9,_,-
 		$request_body ['buffer_id'] = preg_replace( '/[^A-Za-z0-9]/', '', $request_body['buffer_id'] );
```

Comment:
Weird phpcs didn't flag this.
```suggestion
		$request_body['buffer_id'] = preg_replace( '/[^A-Za-z0-9]/', '', $request_body['buffer_id'] );
```